### ISSUE TITLE
Abnormal elbow extension

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5906]
**Preferred term label:** Abnormal elbow extension

**Synonyms**

**Definition (free text, please give PubMed ID)** Limited extension or hyperextension of the elbow.

**Parent term (use hpo.jax.org/app)** Abnormality of the elbow HP:0009811

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be the parent term for Limited elbow extension HP:0001377

**Comment by pnrobinson** [comment]
The terms are already grouped under Limited elbow movement, I do not think this would add anything.
